### ISSUE TITLE
Provide arm64 docker images

### ISSUE THREAD

**Issue by wwwescape** [JMBeresford/retrom#93]
Could you make the Docker images compatible with Raspberry Pi 4? Currently not able to spin up a container to test.

**Comment by JMBeresford** [comment]
Would you happen to know exactly what the constraint is that makes the images incompatible? I do not have a Pi (4 or otherwise) to debug on, but would love to fix the issue if I can get a bit more information.

**Comment by wwwescape** [comment]
My Docker Compose YAML:

```
services:
  retrom:
    image: ghcr.io/jmberesford/retrom-service:latest
    container_name: retrom
    ports:
      - 5101:5101
    environment:
      # IGDB API information, see the metadata-providers section above for more info
      IGDB_CLIENT_ID: eq1bpejr7hpaoxp80iwk5fgfsz0xk8
      IGDB_CLIENT_SECRET: 1y0sroizvxprjk96r460xjvmces0hx

      # using the below example postgres container
      DATABASE_URL: postgres://minecraft_steve:super_secret_password@retrom-db/retrom

      # or, bring your own database:
      # DATABASE_URL: postgres://{db_user}:{db_password}@{db_host}/{db_name}
    volumes:
      - /mnt/storage/Games:/app/library

  # OPTIONAL: spin up a postgres container to use as the database. Or,
  # you can use your own postgres instance by setting DATABASE_URL above
  #
  # read the docs here: https://hub.docker.com/_/postgres
  retrom-db:
    container_name: retrom-db
    hostname: retrom-db
    image: postgres:16
    restart: unless-stopped
    volumes:
      # to store the DB data on the host
      - /home/pi/Docker/Retrom/pg_data:/var/lib/postgresql/data
    environment:
      POSTGRES_USER: minecraft_steve # db user, used to connect to the db
      POSTGRES_PASSWORD: super_secret_password # db password for above user
      POSTGRES_DB: retrom # db name

  retrom-web:
    image: ghcr.io/jmberesford/retrom-web:latest
    container_name: retrom-web
    hostname: retrom-web
    ports:
      - 5102:3000
    environment:
      RETROM_HOST: http://retrom:5101 # the URL of your Retrom server
```

![image](https://github.com/user-attachments/assets/5ffc92e4-df66-41b4-b7cd-2d1adff14342)

**Comment by JMBeresford** [comment]
@wwwescape That image is the [official Postgresql docker image](https://hub.docker.com/_/postgres). I have no control over that, although i know it is common that permissions issues arise for people w/ that image. You can try adding a `user: {uid}:{gid}` directive to the service in your compose file for a user and group ID you know has write permissions for the directory you specified.

For example:

```yml
  retrom-db:
    container_name: retrom-db
    hostname: retrom-db

    # make sure the below has write access to '/home/pi/Docker/Retrom/pg_data'
    user: 1000:1000 # this is a common default

    image: postgres:16
    restart: unless-stopped
    volumes:
      # to store the DB data on the host
      - /home/pi/Docker/Retrom/pg_data:/var/lib/postgresql/data
    environment:
      POSTGRES_USER: minecraft_steve # db user, used to connect to the db
      POSTGRES_PASSWORD: super_secret_password # db password for above user
      POSTGRES_DB: retrom # db name
```

See #95 for another instance of this.

**Comment by JMBeresford** [comment]
Closing for now, as this is seemingly not related to Retrom. Feel free to reach out or re-open if you find other conflicts that are Retrom-related.

**Comment by wwwescape** [comment]
@JMBeresford I don't think its a problem with PostgreSQL. I was able to download that image just fine.

The problems are with the `retrom` and `retrom-web` images.

![image](https://github.com/user-attachments/assets/e6fbe734-1eb6-475c-a9d0-00e6a8bb029d)

**Comment by JMBeresford** [comment]
Ah, I see. Yes, I assume this is due to Retrom only having amd64 images. I can look into building for arm64 as well, I don't think this would take much effort. I'll re-open this.

**Comment by JMBeresford** [comment]
@wwwescape release 0.1.0 should be available for arm64 docker runtimes. Can you confirm you can pull the image now?

**Comment by wwwescape** [comment]
@JMBeresford Thank you! Everything is working swell now and I've got Retrom up and running.

**Comment by wwwescape** [comment]
With the latest changes, I can no longer pull `arm64/v8` images again. Could you please publish them again?

**Comment by JMBeresford** [comment]
@wwwescape thanks for letting me know, fix is incoming but arm builds take quite a while. You can expect this fixed in v0.1.11 which should be available within a few hours

**Comment by JMBeresford** [comment]
Closing this again as I see arm builds being populated. Let me know if you run into additional issues @wwwescape
